### ISSUE TITLE
Cannot Select any entities to inspect in the Object Inspector Pane

### ISSUE THREAD

**Issue by billybarry** [emberjs/ember-inspector#1310]
**Describe the bug**
When trying to use the inspector on the "Data", "Routes", "Components", or "Containers" tabs, I am unable to select an individual model object, component, service, or other entity for inspection.

The UI portion of the select action appears to function properly, as it highlights the selected entity in blue as if it were selected. However, no Object Inspector opens. Clicking on the collapse/hide button manually for that pane will cause the pane to open successfully, with a message that says "No object selected". In the "Routes" tab, I am able to select the inline "$E" to dump a route or controller to the console, but attempting to select an entity causes the same error.

From what I have witnessed, the bug appears to be in the selection functionality itself. The same error appears to surface for all types of failed selections. This bug does not appear to be related to any code changes with my application. It first surfaced (or rather was first noticed) after moving my computer to another position in the room and turning it back on.

When attempting to select an entity for inspection and failing, a warning-level message is piped to the console (below):
```
VM20842:103 Ember Inspector has errored.
This is likely a bug in the inspector itself.
You can report bugs at https://github.com/emberjs/ember-inspector.
Error message: tag[COMPUTE] is not a function
Stack trace: TypeError: tag[COMPUTE] is not a function
    at valueForTag (http://localhost:4200/assets/vendor.js:67642:24)
    at <anonymous>:4795:34
    at Array.forEach (<anonymous>)
    at <anonymous>:4770:24
    at Array.forEach (<anonymous>)
    at calculateCPs (<anonymous>:4769:18)
    at Class.mixinsForObject (<anonymous>:4451:7)
    at Class.sendObject (<anonymous>:4254:26)
    at Class.inspectModel (<anonymous>:840:37)
    at sendEvent (http://localhost:4200/assets/vendor.js:26422:14)
```

**To Reproduce**
Steps to reproduce the behavior for Data Tab:
1. Go to the "Data" tab on the left-hand side of Ember Inspector.
2. Select a model type to open up the list/table view of said model type
3. Select any individual model object.
4. Note the UI looks like it is selected, but there is no Object inspector tab. Note a warning message in the console.

The following reproduction can also be done with any component in the components tab or any route/controller in the routes tab.

**Expected behavior**
Clicking on an entity will allow you to view an object inspector pane for said entity.

**Screenshots**
![Screen Shot 2020-08-14 at 4 58 33 PM](https://user-images.githubusercontent.com/10437961/90295798-7f6f3a80-de4f-11ea-89b5-01b013982f15.png)

**Environment**
- Ember 3.20.4
- Ember Data 3.20
- Ember Inspector 4.1.0
Mac OSX 10.15.6
Chrome 84.0.4147.125
Firefox 79.0

**Additional context**
I am the only developer on my team experiencing these issues, even with all things being the same (repo, branch, docker environment, etc). This leads me to believe something in my environment is causing Ember Inspector to break.

The following troubleshooting steps have already been attempted:
1. Rebuilt application
2. Removed cached node modules, full reinstall of node modules
3. Restarted computer, browser, webpack build process
4. Removed and re-installed Ember Inspector.
5. Tried in Firefox as well, with fresh install of Ember Inspector
6. Applied OS system updates and restarted

I looked through the existing open issues but was unable to find anything that looked like mine. If this is a duplicate I apologize profusely.

**Comment by rwwagner90** [comment]
This was fixed by https://github.com/emberjs/ember-inspector/pull/1299, we just need to cut a new release.

**Comment by billybarry** [comment]
> This was fixed by #1299, we just need to cut a new release.

Awesome, thank you!
